app/waf/src/services/bigip.service.ts

```diff
@@ -258,16 +253,13 @@ export class BigIpManager {
   }
 
   async installDO(): Promise<string> {
-    // create the body with following format.
-    // await this.mustBeReachable();
+    await this.mustBeReachable();
```

Comment:
we have checked if the bigip is reachable or not in the uploadDO. Do we still need to check this in installDO?
If you are intended for the consistence about the coding style, I am also ok with this.

Reply:
Thanks. Let's keep it as it is. Still in finding the unexpected mustReachable issue..